packages/ckeditor5-media-embed/src/ui/mediaformview.ts

```diff
@@ -104,7 +104,6 @@ export default class MediaFormView extends View {
 
 		this.saveButtonView = this._createButton( t( 'Save' ), icons.check, 'ck-button-save' );
 		this.saveButtonView.type = 'submit';
-		this.saveButtonView.bind( 'isEnabled' ).to( this, 'mediaURLInputValue', value => !!value );
 
```

Comment:
Food for thought: maybe this `role="alert"` is incorrect? https://github.com/ckeditor/ckeditor5/blob/037071ef1934721d01b433f85419e748de48c027/packages/ckeditor5-ui/src/labeledfield/labeledfieldview.ts#L279 I suppose it's more likely associated with aria-live instead of form validation https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role.